main.py

```diff
@@ -40,8 +41,12 @@ def revoke_release(repo, version):
 def do_release(repo, build_number, headers):
     function_url="https://us-central1-language-team.cloudfunctions.net/release"
     url=f"{function_url}/{repo}/{build_number}"    
+    separator='?'
     if attach == "true":
-      url=url+"?attach=true"
+      url=url+f"{separator}attach=true"
+      separator='%'
+    if run_rules_cov == 'true':
+      url=url+f"{separator}run_rules_cov=true"
     return requests.get(url, headers=headers)
```

Comment:
I would prefer the usage of the `params` parameter of the `get` function, to get a better maintainability.

Reply:
indeed, let's fix this